NEMO/templates/calendar/calendar.html

```diff
@@ -24,8 +24,14 @@
 		</ul>
 	</div>
 	<div id="expand-collapse" class="btn-group sidebar-item">
-		<button type="button" class="btn btn-default" style="width:50%" onclick="expand_all_categories()" title="Expand all categories"><span class="glyphicon glyphicon-resize-full"></span></button>
-		<button type="button" class="btn btn-default" style="width:50%" onclick="collapse_all_categories()" title="Collapse all categories"><span class="glyphicon glyphicon-resize-small"></span></button>
+        {% if calendar_qualified_tools %}
+            <button type="button" class="btn btn-default" style="width:33%" onclick="expand_all_categories()" title="Expand all categories"><span class="glyphicon glyphicon-resize-full"></span></button>
+            <button type="button" class="btn btn-default" style="width:33%" onclick="collapse_all_categories()" title="Collapse all categories"><span class="glyphicon glyphicon-resize-small"></span></button>
+            <button type="button" class="btn btn-default" id="qualified_tools_btn" style="width:34%" onclick="toggle_qualified_tools()" title="Show qualified tools"><span class="glyphicon glyphicon-lock"></span></button>
```

Comment:
staff members can see and use all tools even when not explicitly qualified.
so I think it might be better to only show the button for regular users.

also, not sure if it'll be an issue, but something to check is if a regular user logs out and a staff member logs in after him on the same computer, it shouldn't hide half the tools. (probably means adding an extra condition before hiding tools to only do it for regular users)

Reply:
I updated the condition to check if the user is staff before performing any filtering operation